src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -152,6 +152,33 @@ const WorkspaceAccessError = () => {
   ])
 }
 
+const useClusterPolling = ({ namespace }) => {
```

Comment:
Since this only has one argument, why not make it just the string?

Reply:
Fair point, I was being overly defensive.